packages/sources/etherscan/test/unit/adapter.test.ts

```diff
@@ -0,0 +1,34 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src/adapter'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+  process.env.API_KEY = process.env.API_KEY ?? 'test_api_key'
+
+  describe('validation error', () => {
+    const requests = [
+      {
+        name: 'empty body',
+        testData: {},
+      },
+      {
+        name: 'empty data',
+        testData: { data: {} },
+      },
```

Comment:
This test will also probably fail, since the adapter should be able to take no params